pkg/webui/components/qr/input/video/index.js

```diff
@@ -43,16 +47,21 @@ const Video = props => {
     if (devices.length && !stream) {
       const ua = navigator.userAgent.toLowerCase()
       const cameras = devices.filter(device => device.kind === 'videoinput')
+      setCameras(cameras)
+      let rearCamera = cameras.find(device => device.label.toLowerCase().includes('back'))
+      if (!rearCamera) {
+        rearCamera = cameras[1]
```

Comment:
I suspect this would crash if the camera only has one camera (?)

Reply:
it's how it was before on line 50. This bit: [it's how it was before on line 50. This bit: https://github.com/TheThingsNetwork/lorawan-stack/blob/8eab0b5275e6cb7593dbb01fdea4574ce76ef252/pkg/webui/components/qr/input/video/index.js#L50](https://github.com/TheThingsNetwork/lorawan-stack/blob/8eab0b5275e6cb7593dbb01fdea4574ce76ef252/pkg/webui/components/qr/input/video/index.js#L50)